test/gui/shared/scripts/bdd_hooks.py

```diff
@@ -127,10 +128,14 @@ def hook(context):
         filename = context._data["title"].replace(" ", "_") + ".png"
         directory = os.environ["GUI_TEST_REPORT_DIR"] + "/screenshots"
 
-        if not os.path.exists(directory):
+        while not (os.path.exists(directory)):
             os.makedirs(directory)
 
-        pb.savev(os.path.join(directory, filename), "png", [], [])
+        try:
+            pb.savev(os.path.join(directory, filename), "png", [], [])
+        except:
+            pb.savev(os.path.join(directory, filename), "png", [], [])
```

Comment:
ignore the screenshot file if it doesn't exists.
```suggestion
        try:
            pb.savev(os.path.join(directory, filename), "png", [], [])
        except:
            pass
```